test/MSVS/vs-14.1-exec.py

```diff
@@ -20,20 +22,18 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-#
-
-__revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
 
 """
 Test that we can actually build a simple program using our generated
-Visual Studio 14.1 project (.vcxproj) and solution (.sln) files
+Visual Studio 14.0 project (.vcxproj) and solution (.sln) files
```

Comment:
Shouldn't this be 14.1 and not 14.0 ? (same in the 14.2 test)

Reply:
I've lost the context (will try to mentally resync later) but I think there's been a common file format for several vs releases. hard to describe this... too many version descriptions involved.